web/components/gantt-chart/blocks/block.tsx

```diff
@@ -36,7 +36,7 @@ export const GanttChartBlock: React.FC<Props> = observer((props) => {
   } = props;
   // store hooks
   const { updateActiveBlockId, isBlockActive } = useGanttChart();
-  const { peekIssue } = useIssueDetail();
+  const { isIssuePeeked } = useIssueDetail();
```

Comment:
just one comment on naming conventions, `isIssuePeeked` sounds like a boolean. `getIsIssuePeeked` sounds like a method that returns a boolean. please rename to that.